src/main/resources/application.yml

```diff
@@ -14,10 +14,37 @@ spring:
   jpa:
     hibernate:
       ddl-auto: validate
-
 scheduler:
   opene:
     cron:
       expression: '0 0 3 * * *'
     name: opene
     lock-at-most-for: PT5M
+  fallout:
+    cron:
+      expression: '0 0 4 * * *'
+    name: fallout
+    lock-at-most-for: PT5M
+falloutreport:
+  recipients:
+    - teamunmasked@sundsvall.se
```

Comment:
Can this (recipient and sender address) be set in Gitlab, instead of here?
(Maybe a dummy here and the real address in Gitlab)

Reply:
Absolutely, I'll change